package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dcos/ui-kit",
-  "version": "1.3.0",
+  "version": "0.0.0-dev+semantically-released",
```

Comment:
why was this changed?

Reply:
yes

Reply:
https://github.com/dcos-labs/ui-kit/pull/198